### PR TITLE
all snapshot tests refactored

### PR DESCRIPTION
**Pull request by juanky201271**

### CODE REVIEW

File: __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+                  </Text>
+                  <Text
+                    style={
+                      {
+                        "color": "rgb(28, 28, 30)",
+                        "opacity": 0.65,
+                      }
+                    }
+                  >
+                    Jan 3, 12:36 am
```
Comment (AloeareV) [inline __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap]: This is time-zone based, this test fails on my machine with a time of three hours later.

File: __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+                  </Text>
+                  <Text
+                    style={
+                      {
+                        "color": "rgb(28, 28, 30)",
+                        "opacity": 0.65,
+                      }
+                    }
+                  >
+                    Jan 1, 8:48 pm
```
Comment (AloeareV) [inline __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap]: As above, this fails as it outputs 11:48 pm on my computer due to time zones.

File: __tests__/__snapshots__/Transactions.Portrait.snapshot.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,694 @@
...
+                </Text>
+                <Text
+                  style={
+                    {
+                      "color": "rgb(28, 28, 30)",
+                      "opacity": 0.65,
+                    }
+                  }
+                >
+                  Jan 3, 12:36 am
```
Comment (AloeareV) [inline __tests__/__snapshots__/Transactions.Portrait.snapshot.tsx.snap]: Time zone as above

File: __tests__/__snapshots__/Transactions.Portrait.snapshot.tsx.snap [hunk 4]
```diff
@@ -0,0 +1,694 @@
...
+                </Text>
+                <Text
+                  style={
+                    {
+                      "color": "rgb(28, 28, 30)",
+                      "opacity": 0.65,
+                    }
+                  }
+                >
+                  Jan 1, 8:48 pm
```
Comment (AloeareV) [inline __tests__/__snapshots__/Transactions.Portrait.snapshot.tsx.snap]: Time zone as above

**Review by AloeareV — DISMISSED** [review]
Tests are working again! All but two tests pass locally, with time zone issues causing the failure.

File: __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap (hunk 1 above)
Comment (juanky201271) [reply __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap]: Yes, I'm on it... I want to put an UTC date there. I hope this works. If you have any idea, please tell me.

File: __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap (hunk 2 above)
Comment (juanky201271) [reply __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap]: same problem.

File: __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap (hunk 1 above)
Comment (juanky201271) [reply __tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap]: T think I got it. All tests pass.

**Comment by juanky201271** [comment]
> Tests are working again! All but two tests pass locally, with time zone issues causing the failure.

Well, I'm using the `moment` library and I mocked it in the tests. I remove `react-moment` because with `moment` is enough.

**Comment by zancas** [comment]
So do all tests now pass?

**Comment by juanky201271** [comment]
It's a first step, snapshots and some unit tests. I'm going to add more unit tests, and the next step is add integration test with probably cypress...

**Comment by juanky201271** [comment]
BTW I missed the test for `send` screen... I'm working on it.

**Comment by juanky201271** [comment]
@zancas @AloeareV This PR is ready for review and merge. Snapshots and basic unit tests.

**Comment by zancas** [comment]
Tests passing!!!   Hooooray!   You have done amazing work!

**Review by zancas — APPROVED** [review]
This PR allows tests to run!

**Pull request merged**